geoai/universat.py

```
@@ -0,0 +1,287 @@
+    "get_tile_embedding",
+    "get_pca_rgb",
+    "universat_train",
+]
+
+# UniverSat repo cache and setup
+UNIVERSAT_CACHE_DIR = os.path.expanduser("~/.cache/geoai/UniverSat")
+_src = os.path.join(UNIVERSAT_CACHE_DIR, "src")
+_repo_ready = False
+
+
+def _setup():
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_setup`

Add return type annotation: `None`

(ANN202)